### DISCUSSION TITLE
[Testing] Atlanta TvHeadend Server

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#189] [Show and tell]
"https://ghostbin.com/gtoxc" Including All Atlanta OTA Ch. (But Not Working in WPCH and Few Channel)

**Comment by MapGuy11** [comment]
Because WPCH went ATSC 3 and probably the server didn't rescan

**Comment by BellezaEmporium** [comment]
Lots of channels are simply unwatchable when being in HD. Poor bandwidth in my opinion.

However SD channels seem to work like a bliss (depends for some channels)

**Reply by MapGuy11** [reply]
> Lots of channels are simply unwatchable when being in HD. Poor bandwidth in my opinion.
> 
> However SD channels seem to work like a bliss (depends for some channels)

HD is smooth as butter. It must be your latency from Atlanta. As I am only in the Northeastern U.S. they don't studder.

**Reply by BellezaEmporium** [reply]
> > Lots of channels are simply unwatchable when being in HD. Poor bandwidth in my opinion.
> > However SD channels seem to work like a bliss (depends for some channels)
> 
> HD is smooth as butter. It must be your latency from Atlanta. As I am only in the Northeastern U.S. they don't studder.

May be.
